css/style.css

```diff
@@ -18,3 +19,14 @@ button {
   font-size: 18px;
   box-sizing: border-box;
```

Comment:
there is a better alternative for this, but it would be good to set this kind of element settings in another file, you could even use one called `all.css` and put all combinations there